### PR TITLE
Add migration folder and middleware.ts to redirect user when env var …

### PR DESCRIPTION
**Pull request by alexbeno**
…is not set

### CODE REVIEW

File: app/opengraph-image.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Generator from "components/features/seo/image-metadata/commons/generator/generator.tsx";
```
Comment (pixelfact) [inline app/opengraph-image.tsx:1]: maybe move to metadata folder ?

**Review by pixelfact — COMMENTED** [review]
move file if applicable

File: app/opengraph-image.tsx (hunk 1 above)
Comment (alexbeno) [reply app/opengraph-image.tsx:1]: this one is a common component that will be for generated all cards inside metadata folder

File: app/[[...slug]]/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import { sharedMetadata } from "../shared-metadata.ts";
+import { ProjectsActions } from "../../actions/Projects/projects.actions.ts";
+import { UsersActions } from "../../actions/Users/users.actions.ts";
+
+type Props = {
+  params: { slug: string[] };
+  searchParams: { [key: string]: string | string[] | undefined };
+};
+
+export async function generateMetadata({ params, searchParams }: Props, parent: ResolvingMetadata): Promise<Metadata> {
```
Comment (pixelfact) [inline app/[[...slug]]/layout.tsx:11]: check if is searchParams is used ?

**Review by pixelfact — APPROVED** [review]

**Pull request merged**